app/src/main/java/com/example/fitappa/View/ViewRoutineActivity.java

```diff
@@ -24,12 +27,16 @@ protected void onCreate(Bundle savedInstanceState) {
         // Retrieve UI
         this.routineLayout = findViewById(R.id.RoutineLayout);
         Button addWorkoutBtn = findViewById(R.id.AddWorkoutBtn);
+        Button back = findViewById(R.id.backButton2);
```

Comment:
would be nicer to make this backBtn (following the convention of other variables of type Button)